README.md

```diff
@@ -125,14 +121,43 @@ Or you can use the following command to install deepchem without gpu support.
 .\scripts\install_deepchem_conda.ps1 -gpu 0 deepchem
 ```
 
-Before activating deepchem envrionment, make sure conda-powershell has been initialized.
-Check if there is a `(base)` before `PS` in powershell. If not, use `conda init powershell`
-to activate it, then:
+Before activating deepchem environment, make sure conda-powershell has been initialized.  
+Check if there is a `(base)` before `PS` in powershell.  
+If not, use `conda init powershell` to activate it, then:
 ```
 conda activate deepchem
 python setup.py install
 nosetests -a '!slow' -v deepchem --nologcapture
 ```
+
+### Install using a Docker with GPU (WIP)
+
+#### Build the image from Dockerfile (WIP)
+
+We prepare for [sample Dockerfiles](https://github.com/deepchem/deepchem/tree/master/docker) to install deepchem from source codes and conda package manager. Please check them!
+
+#### Use the official deepchem image (WIP)

```

Comment:
Do we manage the deepchem image...?  (latest image is published at 2019/05/04)
If we don't have the capability to manage the image, I think this instruction should be removed.

Reply:
We haven't updated it in a while, but we do have the ability to manage the image. Ping me your dockerhob username and I'll give you manager rights for the image